assets/src/scripts/components/SearchBar/SearchBarResult.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+/**
+ * History List.
+ */
+export default class SearchBar_Result extends React.Component {
```

Comment:
I haven't seen underscores in class names in React, is this according to WP Standards?